hamocc/mo_aufr_bgc.F90

```diff
@@ -339,6 +362,24 @@ subroutine aufr_bgc(kpie,kpje,kpke,ntr,ntrbgc,itrbgc,trc,kplyear,kplmon,kplday,o
         write(io_stdo_bgc,*) ' Initialising atmosphere from scratch '
       endif
     endif
+
+      lread_prefsi=.true.
+      if(IOTYPE==0) then
+        if(mnproc==1) ncstat=nf90_inq_varid(ncid,'prefsilica',ncvarid)
+        call xcbcst(ncstat)
+        if(ncstat.ne.nf90_noerr) lread_prefsi=.false.
+      else if(IOTYPE==1) then
+#ifdef PNETCDF
+        ncstat=nfmpi_inq_varid(ncid,'prefsilica',ncvarid)
+        if(ncstat.ne.nf_noerr) lread_prefsi=.false.
+#endif
+      endif
+      if(mnproc==1 .and. .not. lread_prefsi) then
+        write(io_stdo_bgc,*) ' '
+        write(io_stdo_bgc,*) 'AUFR_BGC info: preformed silica not in restart file '
+        write(io_stdo_bgc,*) 'Initialising preformed tracer from scratch'
+      endif
```

Comment:
A post-review comment: I think the option to initialize preformed Si from scratch while the other preformed tracers are read from restart file is potentially dangerous if users are not aware of this feature.

I do see that being able to start from older restart files is useful for now, but I would suggest to remove this option again for release NorESM 2.5

Reply:
Hi @JorgSchwinger , agreed, so keeping it for now. For NorESM2.5, I will check, if there are climatologies available that can be used (I vaguely remember to have seen some).

Reply:
But wouldn't it be important to have the preformed tracers consistent - either all preformed tracers initialized from scratch (set to zero) or all preformed tracers initialized to a climatological estimate?

Reply:
I guess, my thought went along the lines of receiving a better spunup preformed silicate tracer field with the new tests and then provide them with the next release - since the longer the spinup time, the better it represents the model, if I am not mistaken?

Reply:
Maybe this could be implemented as an option - having a climatology of all preformed tracers to be used as an initial condition. I would have this optional, because I don't think it will make sense for higher resolution runs (unless we interpolate from lower resolutions).

Reply:
For this PR, I put this to the to-do-list #340.